docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,152 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
+
+For example, the following query will show the import latency, if this is low in a range of seconds it is fine to update.
+
+```promql
+sum(rate(operate_import_time_seconds_sum{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+/ sum(rate(operate_import_time_seconds_count{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+```
+
+#### Data migration
+
+As we have harmonized indices there are some details Tasklist needs as part of the processes, which were not part of
+existing Operate process indices.
+
+For this reason, we have built a migration application to enhance existing process models. This migration application will
+run asynchronous next to the other applications, until the importers are done.
+
+This is to make sure that all processes created in 8.7 are migrated and enhanced.
+
+The migration application can be found here: `camunda/bin/migrate`
+
+:::note
+If you deploy the Helm Charts, it should automatically be deployed with the Importers.
+:::
+
+#### Users can turn off Importers after completion
+
+As Importers are only necessary at the beginning, due to importing old data / draining the queue of 8.7 events,
+they can be turned off later.
+
+To detect whether importers are done, users can either validate the state in the Elasticsearch/OpenSearch Index:
+
+- `tasklist-import-position-8.2.0_`
+- `operate-import-position-8.3.0_`
+
+If for all entries the `completed` property is set to true, the importing is done.
+
+Alternatively, metrics can be validated via:
+
+- `operate_import_completed`
+- `tasklist_import_completed`
+
+Example query could be: `sum(tasklist_import_completed{namespace="$namespace"}) by (importPositionAlias)`
+
+![metrics-import-complete](./img/import-complete-metrics.png)
+
+#### Custom prefix usage needs extra care
+
+:::important
+This is a requirement if you use an installation with custom prefixes.
+:::
+
+As we have harmonized our index schema as part of the Camunda Exporter development, we have introduced one common index prefix.
+To update indices need to migrate to one common index, so the Camunda Exporter and also REST API and other consumers
+can be configured correctly.
+
+##### Prefix migration
+
+:::warning
+Disk usage will massively increase while the migration is ongoing.
+Make sure you have at least double of disk space free.
+:::
+
+:::note
+While we test and validate the provided migration script,
+it is recommended that you first back up the Elasticsearch, and test the migration script in a test environment first
+to ensure there are no unexpected issues.
+:::
+
+:::note
+The prefix migration involves downtime.
+:::
+
+Steps for prefix migration:
+
+1. Shut down your Camunda 8.7 cluster, fex. reduce the replicas to zero. Such that no traffic will executed against your to-be-migrated ES/OS prefixes.
+2. In the `camunda/bin` directory there is a `prefix-migration` script that will execute the prefix migration,
```

Comment:
Where is that instance running if the cluster is already shut down?

Reply:
Do you mean the `prefix-migration` application? Can be everywhere, locally or deployed. But in this example locally I would suspect.

Reply:
In that case, let's mention it explicitly to avoid any confusion from the reader.

Reply:
I did in my last commit. :)

Reply:
https://github.com/camunda/camunda-docs/pull/5023/commits/3040ddde6e48b683486300cced3aaea6366ba156